spark/src/main/scala/geotrellis/spark/io/avro/codecs/TileCodecs.scala

```diff
@@ -16,12 +17,14 @@ trait TileCodecs {
       .fields()
       .name("cols").`type`().intType().noDefault()
       .name("rows").`type`().intType().noDefault()
+      .name("celltype").`type`().stringType().noDefault()
```

Comment:
This will need a default for us to be able to read existing records, which are all constant no data types